guides/doc-Configuring_Load_Balancer/topics/Configuring_Capsule_Server_with_Default_SSL_Certificates_for_Load_Balancing_with_Puppet.adoc

```diff
@@ -68,21 +68,41 @@ root@_{smart-proxy-context}_ca.example.com_:__{smart-proxy-context}_ca.example.c
 --enable-foreman-proxy-plugin-remote-execution-ssh
 ----
 
-. On {SmartProxyServer}, generate Puppet certificates for all other {SmartProxyServer}s that you configure for load balancing, except this first system where you configure Puppet certificates signing:
+. On {SmartProxyServer}, generate Puppet certificates for all other {SmartProxyServer}s that you configure for load balancing, except the first system where you configure Puppet certificates signing:
+.. First, stop the Puppet server:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# puppet cert generate _{smartproxy-example-com}_ \
---dns_alt_names=_loadbalancer.example.com_
+# puppet resource service puppetserver ensure=stopped
 ----
 +
+.. Generate certificate for the {SmartProxyServer}:
```

Comment:
This step should be more descriptive. I think this can be fixed by getting rid of substeps - using `.` for steps to keep them in the procedure and not create an additional indentation. With that, we can make this step extra clear and use the full sentence ` . On {SmartProxyServer}, generate Puppet certificates for all other {SmartProxyServer}s that you configure for load balancing, except the first system where you configure Puppet certificates signing:`. What do you think?

Reply:
Updated, thanks.